protected/views/patient/_130_diagnoses.php

```diff
@@ -130,7 +130,7 @@
 							</div>
 						</fieldset>
 
-						<?php $this->renderPartial('_fuzzy_date', array('form' => $form, 'label' => 'Date diagnosed'))?>
+						<?php $this->renderPartial('_fuzzy_date', array('form' => $form, 'label' => 'Date diagnosed', 'empty' => true))?>
```

Comment:
I tried not to have the variable name be too large. I suppose it could be, but technically empty years are always allowed; they just aren't defaulted to (I think)